high_card.rb

```diff
@@ -1,5 +1,31 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
 players = []
+deck = []
+hand = []
+
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank, suit])
+  end
+end
+
+shuffled = deck.shuffle
+
+puts players.length.to_s + " players so far. Enter a player name, or type 'play'"
```

Comment:
use string interpolation instead of all of the + signs and quotes? might also help you get rid of the [""] issues